components/ui/icons/src/main/res/drawable/mozac_ic_private_browsing.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="16"
+    android:viewportHeight="16">
```

Comment:
All other icons seem to use a viewport width/height of 24. Is this intentional?

Reply:
Interesting, I see some with 16 as well, but I'll change it to 24 if that's our standard.